company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise6Test.java

```diff
@@ -62,7 +67,9 @@ public void sortedTotalOrderValue()
     @Test
     public void sortedTotalOrderValueUsingPrimitives()
     {
-        MutableDoubleList sortedTotalValues = null;
+        MutableDoubleList sortedTotalValues = ListIterate
```

Comment:
Hi @nsardana-bny , this should call the method directly on `this.company.getCustomers()` instead.